libfastpm/cosmology.c

```diff
@@ -273,33 +272,44 @@ static ode_soln growth_ode_solve(double a, FastPMCosmology * c)
     gsl_odeiv2_driver * drive 
         = gsl_odeiv2_driver_alloc_standard_new(&F,
                                                gsl_odeiv2_step_rkf45, 
-                                               1e-5, 
+                                               1e-6,
                                                1e-8,
                                                1e-8,
                                                1,
                                                1);
     
      /* We start early to avoid lambda, but still MD so we use y<<1 Meszearos soln. And assume all nus are radiation*/
-    //double aini = 1e-5;
     
     //Note the normalisation of D is arbitrary as we will only use it to calcualte growth fractor.
     
-    //MD initial conditions for now.
-    //double yini[4] = {aini, aini, -3./7.*aini*aini, -6./7.*aini*aini};           // FIXME: these are wrong. See Yu and Jia paper 2018
+    // ASSUME MD, no FS
+    //double aini = 4e-2;
```

Comment:
Let's at least add a space between // and text. 
If you are commenting out a large chunk of code, then using #ifdef 0 #endif is a typical pattern (vim knows to highlight it as comment).